image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -406,33 +406,45 @@ def image_patching(nuage_config):
 
 def check_config(nuage_config):
     missing_config = []
-    for key in ["ImageName", "DeploymentType", "RepoFile", "VRSRepoNames"]:
+    for key in ["ImageName", "RepoFile", "VRSRepoNames"]:
         if not (nuage_config.get(key)):
             missing_config.append(key)
     if missing_config:
         logger.error("Please provide missing config %s value "
                      "in your config file. \n" % missing_config)
         sys.exit(1)
-    if "avrs" in nuage_config["DeploymentType"] and "ovrs" in nuage_config["DeploymentType"]:
+    file_exists(nuage_config["ImageName"])
+
+    if not nuage_config["DeploymentType"]:
+        logger.info("Overcloud Image will be patched with Nuage VRS rpms")
+    elif "avrs" in nuage_config["DeploymentType"] and "ovrs" in nuage_config["DeploymentType"]:
         logger.error(
             "Currently Nuage doesn't support both AVRS and OVRS deployment together"
             "Please choose only one between them")
         sys.exit(1)
-    if "avrs" in nuage_config["DeploymentType"]:
+    elif nuage_config["DeploymentType"] is "avrs":
```

Comment:
I thought of the same "if, elif and else", but what if we want to support both AVRS + OVRS?

Reply:
Not needed anymore